OJP/OJP_ModesSupport.xsd

```diff
@@ -1,55 +1,189 @@
+			</xs:enumeration>
+			<xs:enumeration value="commuterPooling">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION. A pooling to go to workplaces and related things. Usually in a regular interval with the same people.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="parkAndRidePooling">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION. A pooling in relation to park and ride.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="chauffeured">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION for SINGLE JOURNEY with a paid driver. Often more than a single SINGLE JOURNEY.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="demandResponsive">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION demand responsive is used, when there are no SCHEDULED STOP POINTs involved and the timetable component is weak.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="flexibleArea">
+				<xs:annotation>
+					<xs:documentation>Specialisation of the demand responsive ALTERNATIVE MODE OF OPERATION for AREA related offers.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="school">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION. To specifically state that it is school related. Specialisation of demand responsive.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="companyShuttle">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION. To specifically state that it is a company related. Specialisation of demand responsive.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="hotelShuttle">
+				<xs:annotation>
+					<xs:documentation>ALTERNATIVE MODE OF OPERATION. To specifically state that it is hotel related. Specialisation of demand responsive.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="other">
+				<xs:annotation>
+					<xs:documentation>Only use this value, when no other applies.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:annotation>
+		<xs:documentation>==========================================IndividualTransportOptionsStructure ==========================================</xs:documentation>
+	</xs:annotation>
 	<xs:complexType name="IndividualTransportOptionsStructure">
 		<xs:annotation>
 			<xs:documentation>Individual modes and their usage limits as stated by the passenger (TRIP ACCESS CONSTRAINT).</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="Mode" type="IndividualModesEnumeration">
+			<xs:element name="PersonalModeOFOperation" type="PersonalModesOfOperationEnumeration" minOccurs="0">
```

Comment:
The annotation will need to be updated, depending on the enum values we'll have in the end.